pkg/version/version.go

```diff
@@ -29,7 +27,6 @@ func getCollectorVersion() string {
 
 // Versions represents a collection of various pieces of version information.
 type Versions struct {
-	BuildDate time.Time `json:"BuildDate"`
```

Comment:
I'd be interested to know what research you've done and what you found.

Reply:
>  I suppose it was mostly informational
That was my feeling as well. 

I mostly search thru our own code and github using sourcegraph (unfortunately buildDate is a common name and appears in may repositories even in k8s). My reasoning why removing it should be safe if that it's not stored as protobuf and not tested so I thought it should be safe to remove. 
I was thinking about setting a constant value like unix epoch start but I think it's better to not provide information then providing a wrong one.

> Can we safely drop that? 
I think yes but how about adding a deprecation notice and remove it in future release so we will be safe and users informed.

Reply:
> how about adding a deprecation notice and remove it in future release so we will be safe and users informed.

That'll be the safest. It would be required to trace which APIs expose that (besides roxctl) to properly prepare users for this change.